include/controllers/game_controller.hpp

```diff
@@ -22,7 +22,9 @@ class GameController {
     SceneManager<GameState>& getSceneManager() const;
     int getScreenWidth() const;
     int getScreenHeight() const;
+    void loadLobby();
     void startGame();
+    void exitGame();
```

Comment:
I could not test this unfortunately, because my ccmake .. makes my *make*s fail. Do any memory leaks occur on the game side, so no drivers or third party libraries included.

Reply:
According to @Peter-JanGootzen , not on the current master. Not sure about this branch tho